src/sentry/ingest/transaction_clusterer/rules.py

```diff
@@ -152,6 +155,22 @@ def get_sorted_rules(project: Project) -> List[Tuple[ReplacementRule, int]]:
 
 
 def update_rules(project: Project, new_rules: Sequence[ReplacementRule]) -> None:
+    # Run the updates even if there aren't any new rules, to get all the stores
+    # up-to-date.
+
+    last_seen = _now()
+    new_rule_set = {rule: last_seen for rule in new_rules}
+    rule_store = CompositeRuleStore(
+        [
+            RedisRuleStore(),
+            ProjectOptionRuleStore(),
+            LocalRuleStore(new_rule_set),
+        ]
+    )
+    rule_store.merge(project)
+
+
+def update_redis_rules(project: Project, new_rules: Sequence[ReplacementRule]) -> None:
```

Comment:
`update_rules` and `update_redis_rules` look almost the same, but I couldn't come up with a way to use a helper method that also makes the code easier to read.